.ci/docker/ci_commit_pins/executorch.txt

```diff
@@ -1 +1 @@
-d4b3e5cc607e97afdba79dc90f8ef968142f347c
+172574a6be5910a4609e4ed1bef2b6b8475ddb3d
```

Comment:
Curious how do we update this regularly?

Reply:
We have a bot to update this nightly on PT https://github.com/pytorch/pytorch/pull/123043.  The whole process is automated and the new commit is landed once ET build and test pass.  However, as you can see that the build has been failing for a while.  After this PR fixes ET build, future commit hash update will work.  On the other hand, I think we might want to setup an alert of sort here to have someone take a look when PRs like https://github.com/pytorch/pytorch/pull/123043 keep failing.